src/mainwindow.py

```diff
@@ -375,6 +376,37 @@ def on_action_export_troxel_triggered(self):
         tl = TroxelLink()
         webbrowser.open(tl.export(), 2)
 
+    @QtCore.Slot()
+    def on_action_copy_selection_to_frame_triggered(self):
+        target_frame, res = QtGui.QInputDialog.getInt(
+            self, "Copy selection", "Copy selection to frame:", 1, 1, self.display.voxels.get_frame_count())
+        if res:
+            target_frame -= 1
+            original_frame = self.display.voxels.get_current_frame()
+            original_selection = copy.deepcopy(self.display.voxels._selection)
+            if target_frame != original_frame:
+                stamp = []
+                for x, y, z in self.display.voxels._selection:
+                    col = self.display.voxels.get(x, y, z)
+                    stamp.append((x, y, z, col))
+                self.display.voxels.select_frame(target_frame)
+                if self.display.voxels.is_free(stamp):
+                    for x, y, z, col in stamp:
+                        self.display.voxels.set(x, y, z, col)
+                else:
+                    result = QtGui.QMessageBox.question(self, "Copy selection",
+                                                        "This would override voxel data in the targeted frame!",
+                                                        QtGui.QMessageBox.StandardButton.Abort |
+                                                        QtGui.QMessageBox.StandardButton.Ignore)
+                    if result == QtGui.QMessageBox.Ignore:
+                        for x, y, z, col in stamp:
+                            self.display.voxels.set(x, y, z, col)
```

Comment:
Maybe something like this to reduce code duplication:
```python
                btns = QtGui.QMessageBox.StandardButton.Abort | QtGui.QMessageBox.StandardButton.Ignore
                if (self.display.voxels.is_free(stamp) or
                    QtGui.QMessageBox.question(self, "Copy selection",
                                                     "This would override voxel data in the targeted frame!",
                                                     btns) == QtGui.QMessageBox.Ignore):
                    for x, y, z, col in stamp:
                        self.display.voxels.set(x, y, z, col)
```